_sass/module/_landing-description.scss

```diff
@@ -3,15 +3,19 @@
   color: $color-grey-2;
   font-style: italic;
   font-weight: 400;
+  text-align: center;
+  margin: auto;
+  margin-bottom: 5px;
```

Comment:
Maybe use `$spacing-base` here instead of pixel values?

Reply:
Done, thanks!